app/presenters/dttp/trainee_presenter.rb

```diff
@@ -2,8 +2,10 @@
 
 module Dttp
   class TraineePresenter
-    FEMALE_GENDER_CODE = 1
-    MALE_GENDER_CODE = 2
+    MALE_GENDER_CODE = 1
+    FEMALE_GENDER_CODE = 2
+    OTHER_GENDER_CODE = 98
+    GENDER_NOT_PROVIDED_CODE = OTHER_GENDER_CODE
```

Comment:
I took a sample of the `/contacts` API and could only see 4 possibilties for the `gendercode` property: 1, 2, 389040000 and `null`. Maybe `null` means gender not provided?

Reply:
I wondered if that might be the case re: `null`. `Not provided` is set by it 'not being provided'. I'll put this on the BAU catchup questions. This is probably ok for now and we can fix when we hear back from them. Are you also saying that `98` might not be right @jhenderson

Reply:
This has been reviewed offline. The codes are:

1 Female
2 Male
389040000 Other

The BAU back office system submits -1 for not provided as far as we can see. I suspect -1 will actually be persisted rather than being set to `null`. If so we should try sending `null`.

Reply:
updated the codes in https://github.com/DFE-Digital/register-trainee-teachers/pull/356/commits/33e069ffa448f87d6af226ec41af1bd5e425164d